presto-product-tests/src/main/resources/sql-tests/testcases/window_functions/windowDescriptionAABwithEP.sql

```diff
@@ -0,0 +1,8 @@
+-- database: presto; groups: window;
+select
```

Comment:
They may not be overly useful, but they wouldn't be rendered invalid. What's special about lead, lag, ntile, and nth_value is that they have additional parameters, not that those additional parameters end up on a projection node.